### PR TITLE
README update with corrections and include DCR as requirement

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

This updated the readme, notably

- Correct the contents layout
- Correct dead links
- Improve the section on installing and using nvm
- Explain that DCR is now a requirement to be serving article content in frontend (and show how to use it).

### CODE REVIEW

File: docs/01-start-here/01-installation-steps.md [hunk 1]
```diff
@@ -53,15 +63,15 @@ You need a Mac or Linux PC (Ubuntu).
     $ cd frontend
     ```
 
-1. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend* and *CMS fronts* (you will need access to Janus)
+1. Get AWS Credentials using [Janus](https://janus.gutools.co.uk/) for *Frontend*, *CMS fronts* and *CAPI* (You will need access to Janus).
```
Comment (marjisound) [inline docs/01-start-here/01-installation-steps.md:66]: Do you mean Content API for CAPI? In Janus to get the credential it's called `Content API`

**Review by marjisound — APPROVED** [review]

**Comment by marjisound** [comment]
That's really good @shtukas It's easy enough for any new joiner to find their way through it. Thanks

**Review by nicl — COMMENTED** [review]
My only thing to improve is to perhaps make a bit clearer that if you are just developing on article layout etc. you are probably better off developing on DCR (and then connecting to Frontend if required) rather than the other way around. So that people don't get stuck always using Frontend when they don't have to.

I agree that it is difficult to summarise this clearly to a newby because what DCR supports is a moving target.

File: docs/01-start-here/01-installation-steps.md (hunk 1 above)
Comment (shtukas) [reply docs/01-start-here/01-installation-steps.md:66]: Yeah, I probably did. Sorry I was quoting from bad memory...

**Comment by shtukas** [comment]
@nicl I made the update you requested. Cheers

**Review by nicl — APPROVED** [review]

**Pull request merged**